scripts/install2

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-TAG_CJDNS=d2e55d58548d83940482fe1bbbe1fd36f7f1b4ef
+TAG_CJDNS=a86c3e147b5e3eefbbe2c94c8c5aded57e3dfbaf
```

Comment:
We can just put human-readable tags like `v20.2` instead of hashes. If I remembered correctly, we ended up at a hash at some point because we wanted to pull in a pre-release patch.

Reply:
Ahh nevermind is this just to test? The goal is we move to 20.2 right?